python/cugraph/cugraph/structure/replicate_edgelist.py

```diff
@@ -0,0 +1,150 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import dask_cudf
+import cudf
+from dask.distributed import wait, default_client
+from pylibcugraph import (
+    ResourceHandle,
+    replicate_edgelist as pylibcugraph_replicate_edgelist,
+)
+
+from cugraph.dask.common.part_utils import (
+    get_persisted_df_worker_map,
+    persist_dask_df_equal_parts_per_worker,
+)
+
+import dask
+import cupy as cp
+import cugraph.dask.comms.comms as Comms
+from typing import Union, Tuple
+
+edgeWeightCol = "weights"
```

Comment:
Is there a global structure that captures the standard column names so that each files does not need to define “src” each time and we have no conflict with “weight” vs “weights”?

Reply:
I just pushed a commit that removed the hardcoded column names and allows the user to provide any column names.